command_line/powder_calibrate.py

```diff
@@ -349,20 +363,23 @@ def __init__(
         self.detector = start_geometry.detector
         self.calibrant = calibrant
         self.coarse_geom = coarse_geom
+
         self.fig, self.ax = self.set_up_figure()
         self.calibrant_image = self.calibrant_rings_image()
-        self.beam_fast_slider = self.make_slider("fast")
-        self.beam_slow_slider = self.make_slider("slow")
+
+        self.beam_fast_slider = self._make_slider("fast")
+        self.beam_slow_slider = self._make_slider("slow")
+        self.distance_slider = self._make_slider("distance")
 
     def __repr__(self):
         calibrant = os.path.splitext(os.path.basename(self.calibrant.filename))[0]
-        return f"Eyeballing Widget using {calibrant} Calibrant starting from Geometry: \n {self.geometry}"
+        return f"Eyeballing Widget instance using {calibrant} Calibrant starting from Geometry: \n {self.geometry}"
```

Comment:
```suggestion
        return f"<Eyeballing Widget instance using {calibrant} Calibrant starting from Geometry: \n {self.geometry}>"
```
Is this used for direct output anywhere? I tend to try for this sort of `__repr__` to vaguely follow python's plain object convention, it can be hard to separate objects in the terminal otherwise.
```
>>> object()
<object object at 0x1028288a0>
```
If it's actually used for user-output anywhere then it's fine as-is.

Reply:
Ooh, yes good call!